user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Maps/MapsTutorial.md

```diff
@@ -0,0 +1,171 @@
+
+1. In the *Layout* tab, go to the *Map settings* section, and click *Save current view*.
+
+   This will set the default map center and default zoom level to what is currently displayed.
+
+1. Under *Map zoom*, use the slider to limit the minimum zoom level.
+
+   This will make it impossible for a user to zoom out past a certain level.
+
+1. Under *Map bounds*, select *Enable bounds*.
+
+   This will limit the bounds of the map, so that the user cannot pan too far away away from the markers.
+
+   ![MapsSettings](~/user-guide/images/MapsSettings.png)
+
+## Step 4: Style the markers
+
+At this point, the cell tower markers are always shown in the component, but they still look very basic. To make them stand out more and display some additional information, you can edit their template:
+
+1. In the *Layout* tab for the component expand the *Layer settings* and then expand the *Cell towers* layer.
+
+1. In the *Template* box, click *Edit*.
+
+   This will open the template editor.
+
+1. Create a text layer that displays the number of transceivers on the tower:
+
+   1. In the *Tools* tab on the left, select *Text* and then click and drag in the center pane to add a text layer.
+
+   1. In the box below *Show text* on the right, enter the text `{Tranceivers}`.
+
+      This placeholder text will be replaced by the content of the corresponding cell value from the query data source.
+
+1. Create an ellipse layer as the background for the text:
+
+   1. In the *Tools* tab on the left, select *Ellipse* and then click and drag to create an ellipse shape around the text placeholder in the center pane.
+
+   1. Below *Show ellipse* on the right, click the color icon to select a custom color for the background.
+
+1. Create an icon layer to represent the cell towers:
+
+   1. In the *Tools* tab on the left, select *Ellipse* and then click and drag to place the icon in the right location in the center pane.
+
+   1. In the pane on the right, click *Icon*, enter `tower` in the filter box, and select the tower icon.
+
+The result should look like this:
+
+![CellTowersTemplate](~/user-guide/images/MapsCellTowersTemplate.png)
+
+![CellTowersTemplateResult](~/user-guide/images/MapsCellTowersTemplateResult.png)
+
+## Step 5: Add grouped markers
+
+The templates you added in the previous step look good, but at this point the visualization still feels cluttered when you zoom out all the way. You can deal with this by adding grouped markers. For this, you will need to create additional queries to retrieve data from the *CellTowersGroups* and *CellTowersCities* JSON files you installed with the package in step 1:
+
+To visualize these we need 2 new queries identical to our first one, but with a different file as a source. Theses queries can then be added to the maps component. We also configure a simple template for both queries that shows the *Count* of cell towers that are grouped in the markers.
```

Comment:
From this line onward (also for the steps below), could you detail what the user should do step by step using a numbered list, like I did in the reviewed sections above? Each numbered list item should represent an action. The result of an action can be described in an indented line under the list item when necessary. This way, it will be easier for an inexperienced user to follow the tutorial, because right now you're going over everything so quickly that they're likely to get lost. Also, to draw the reader into the text, it's best to address them directly, so please use the "you" perspective, rather than the "we" perspective (like I did in the reviewed section above).